language/move-compiler/src/inlining/translate.rs

```diff
@@ -83,11 +85,12 @@ impl<'l> Inliner<'l> {
                 if !fdef.inline {
                     let mut visitor = OuterVisitor { inliner };
                     Dispatcher::new(&mut visitor).function(fdef);
+                    //Self::eprint_fdef(&format!("fun {} ", _name), fdef);
```

Comment:
```suggestion
```

Reply:
Prefer to keep -- as discussed for last PR.